spdm_dump/spdm/spdm_dump_spdm.c

```diff
@@ -2999,11 +3058,37 @@ void dump_spdm_chunk_response(const void *buffer, size_t buffer_size)
                spdm_response->header.param2,
                spdm_response->chunk_seq_no,
                spdm_response->chunk_size);
+
         if (spdm_response->chunk_seq_no == 0) {
+            /*first chunk*/
             large_message_size = (void *)(spdm_response + 1);
             printf(", LargeMsgSize=0x%08x", *large_message_size);
+            responder_send_large_message = malloc(*large_message_size);
+            final_response_message_size = *large_message_size;
+            store_response_large_message = responder_send_large_message;
+            /*point to SPDMchunk data*/
+            ptr = (uint8_t *)(spdm_response + 1) + sizeof(uint32_t);
+        } else {
+            /*point to SPDMchunk data*/
+            ptr = (void *)(spdm_response + 1);
+        }
+
+        /*store chunk data to large message*/
+        memcpy(store_response_large_message, ptr, spdm_response->chunk_size);
+
+        /*move to store next chunk*/
+        store_response_large_message += spdm_response->chunk_size;
+
+        /*last chunk: dump total large message and free requester_send_large_message*/
+        if (spdm_response->header.param1 == SPDM_CHUNK_SEND_REQUEST_ATTRIBUTE_LAST_CHUNK) {
+            printf("\ntotal large message data =\n");
+
```

Comment:
please check "Segma (ChunkSize) == LargeMessageSize"

Reply:
Thanks.  I have added the check in last_chunk.
```
            /*final total chunk size shall equal the LargeMessageSize*/
            if (chunk_send_large_message_current_size != chunk_send_large_message_buf_size) {
                free(chunk_send_large_message_buf);
                chunk_send_large_message_buf = NULL;
                printf("\n");
                return;
            }
```